scripts/code-health.ps1

```
@@ -0,0 +1,55 @@
+            if ($line -match $p.regex) {
+                $col = ([regex]::Match($line, $p.regex)).Index + 1
+                $item = [pscustomobject]@{
+                    file = $file.FullName
+                    line = $idx
+                    column = $col
+                    pattern = $p.name
+                    severity = $p.severity
+                    text = $line.Trim()
+                }
+                $issues += $item
+                Write-Host "[ISSUE] $($file.FullName):$idx:$col - $($p.name): $($line.Trim())" -ForegroundColor Yellow
```

PSScriptAnalyzer (1.23.0)

[error] Variable reference is not valid. ':' was not followed by a valid variable name character. Consider using ${} to delimit the name.

(InvalidVariableReferenceWithDrive)